apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/SuccessMessage.tsx

```diff
@@ -22,17 +22,18 @@ export const SuccessMessage = ({ environment, survey, webAppUrl, user }: Summary
   const [confetti, setConfetti] = useState(false);
 
   const isAppSurvey = survey.type === "app" || survey.type === "website";
+  const widgetSetupCompleted = environment?.appSetupCompleted || environment?.websiteSetupCompleted;
```

Comment:
Can we make these checks more accurate?
I would prefer if we could check depending on the survey type:
```ts
const widgetSetupCompleted = survey.type === "app" ? environment?.appSetupCompleted : environment?.websiteSetupCompleted;